CONTRIBUTING.md

```diff
@@ -40,51 +38,55 @@ You need to fork [scratchy repository](https://github.com/mdl29/scratchy)
 1. Choose a implicite title (max ~ 13 words),
     your title must be begin by a tag.
     > tags have same name of labels (Front,Back,Doc,Bug,Help...)   
-    * **example :** `[Front] Create scratchy service that contains methods for backend requests access`
-    ![example issue](doc/example/example-issue.png)
-2. write a good description with much details as possible 
+
+2. Write a good description with much details as possible 
 
 3. Assign your issue to yourself if you want to solve it or wait for someone to assign it
 
 4. Add labels, for example in scratchy service issue (picture) we use frontend and enchancement labels.
 **see all labels [here](https://github.com/mdl29/scratchy/labels) and their meaning**
 
-4. Add your issue to a project the current one ([V1](https://github.com/mdl29/scratchy/projects/1)) if it is important or the next one if not
+5. Add your issue to a project the current one ([V1](https://github.com/mdl29/scratchy/projects/1)) if it is important or the next one if not
 because in the future we will go over [V2](https://github.com/mdl29/scratchy/projects/2)
  
-5. Add your issue to the current one milestone if it is important or the next one.
+6. Add your issue to the current one milestone if it is important or the next one.
+
+* **Example :** `[example] Create scratchy service that contains methods for backend requests access`
+    ![example issue](doc/example/example-issue.png)
   
 
 ## **Create a pull request :**  
 
 When you think your code is ready to be merged to the project create a pull request.
 
-1. choose the title:
-    when your pull request is ready link to an issue :
-
-    * **example :**` Resolves #ISSUEID : title of the issue `
+1. Choose the title:
+    patern: `ISSUEID title of the issue `
```

Comment:
I think is better `Resolves #ISSUEID : title of the issue` or a title like this.
It's more design xd

Reply:
just putting resolves in front of every pr don't add anything
i prefer to stick with something closer to the [git convention](https://namingconvention.org/git/pull-request-naming.html)

Reply:
hmm i think is needed to request @Benvii to choose the format

Reply:
I propose my own, self-made convention: the DWTFYWALAIRAUNOGASA(EY)
Do
Whatever
The
Fuck
You
Want
As
Long
As
It's
Readable
And
Understandable,
No
One
Gives
A
Shit
Anyways
(
Except
You
)

Reply:
i think you could improve the name but it seem to be an interesting proposition
could you make a linter to help people use it :joy: